src/components/spinner.tsx

```diff
@@ -0,0 +1,21 @@
+import clsx from 'clsx';
+import { FC } from 'react';
+
+export const Spinner: FC<{ className?: string }> = ({ className }) => (
+  <div
+    className={clsx(className, 'animate-spinnerContainer')}
+    role="progressbar"
+  >
+    <svg className="w-full h-full" viewBox="22 22 44 44">
+      <circle
+        className="animate-spinnerIndicator"
+        cx="44"
+        cy="44"
+        r="20.2"
+        fill="none"
+        stroke="currentColor"
```

Comment:
huh i don't think i knew about `currentcolor`!

Reply:
It becomes whatever the text color is at this point in the DOM, if the SVG is part of the DOM. (If the SVG is loaded via an img tag, currentColor is just black.)